docs/user_guide.md

```diff
@@ -237,6 +237,7 @@ $ docker-compose start
 For more information about GC, please see [GC](https://github.com/docker/docker.github.io/blob/master/registry/garbage-collection.md).  
 
 ### Content trust  
+**CAUTION: notary is an optional installing component, please make sure you have already enable it in your Harbor before you go through this part!**  

```

Comment:
NOTE: Notary is an optional component, please make sure you have already installed it in your Harbor instance before you go through this section.